### PR TITLE
[WFLY-12800] Missing documentation for Weld and Microprofile capabilities

### PR DESCRIPTION
**Pull request by yersan**
Jira issue: https://issues.jboss.org/browse/WFLY-12800

### CODE REVIEW

**Comment by yersan** [comment]
@bstansberry FYI: The links to this capabilities are used by wildscribe documentation, which is in turns distributed on the product (including Beta)

File: org/wildfly/weld/capability.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+DESCRIPTION: Provides an API that exposes the functionalities available on the Weld subsystem to the deployment units. E.g., the ability to check if a deployment unit was marked as Weld Deployment, the ability to register a CDI extension for a deployment unit.
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: true
```
Comment (bstansberry) [inline org/wildfly/weld/capability.adoc:12]: This is not private.

File: org/wildfly/microprofile/config/capability.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+DESCRIPTION: The presence of this capability in the capability registry indicates the presence of org.wildfly.extension.microprofile.config-smallrye subsystem in the server.
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: true
```
Comment (bstansberry) [inline org/wildfly/microprofile/config/capability.adoc:12]: This is not private.

**Comment by yersan** [comment]
I think I completely misunderstood what makes a capability public/private, what makes that? the ability to get the service from a caller module? the capability declaration modifier (public, private, package visibility?

**Comment by yersan** [comment]
The description about privacy is in the readme of this project, so, forget my previous questions

**Pull request merged**